orca-spring-batch/src/main/java/com/netflix/spinnaker/orca/batch/SpringBatchActiveExecutionTracker.java

```diff
@@ -95,15 +104,16 @@ public SpringBatchActiveExecutionTracker(JobOperator jobOperator,
   }
 
   /**
-   * Records the count of active executions running on _this_ instance.
+   * Records the details of active executions running on _this_ instance.
    */
   @Scheduled(fixedDelay = FREQUENCY_MS)
   public void countRunningExecutions() {
```

Comment:
Would probably help future readers if we rename the various count* methods (and maybe tests too).